controls/treeview/templates/item-template.md

```diff
@@ -2,19 +2,23 @@
 title: Item Template
 page_title: .NET MAUI TreeView Documentation - Item Template
 description: Learn how to customize the default text displayed in the Telerik .NET MAUI TreeView control by using the item template. 
-position: 7
+position: 1
+previous_url: /controls/treeview/item-template
 slug: treeview-item-template
 ---
 
 # .NET MAUI TreeView Item Template
 
-The TreeView `ItemTemplate` property enables you to customize the text displayed in the TreeView items. Using this property allows you to apply text styles to all TreeView items. The property can be set on TreeView level and/or `TreeViewDescriptor` level.
+The TreeView `ItemTemplate` property enables you to customize the text displayed in the TreeView items. Using this property allows you to add additional elements to the text in the TreeView item. The property can be set on TreeView level and/or `TreeViewDescriptor` level.
 
-In addition to the item's text, TreeView allows you to customize the checkbox, image, and expand/collapse indicator. For this purpose, use the [Control Template]({%slug treeview-control-template%}).
+In addition to the item's text, TreeView allows you to customize the checkbox, image, and expand/collapse indicator. If you want to replace the entire template of the TreeViw check the TreeView [Control Template]({%slug treeview-control-template%}) article. 
 
-The following example shows how to define custom `ItemTemplate` by using the `ItemTemplate` property: 
+The examples below shows how to define custom `ItemTemplate` by using the `ItemTemplate` property.
 
-**1.** Define the `ItemTemplate` on the `RadTreeView` level: 
+### Example with ItemTemplate on RadTreeView level: 
```

Comment:
- Never skip heading levels.
- There's no need to repeat "example" in every heading. Be direct.

```suggestion
## Defining an ItemTemplate on RadTreeView Level 
```